docs/UserGuide.adoc

```diff
@@ -366,7 +366,7 @@ Deletes a person with the NRIC S98654322D in ORGANice.
 === Detecting matches: `match`
 
 This command runs a kidney matching test on patients and donors in ORGANice and displays the match results.
-Refer to <<FAQ>> for a better understanding of the procedure for kidney matching.
+Refer to <<Glossary>> for a better understanding of the procedure for kidney matching.
```

Comment:
FAQ seems correct here why change to `Glossary`

Reply:
Amended.